app/views/documents/index.html.erb

```diff
@@ -131,10 +131,15 @@
             <p class="govuk-body govuk-!-margin-bottom-1">
               <%= link_to image_tag(document.file.representation(resize: "300x212")),
                           url_for_document(document), target: :_blank %>
+            </p>
             <p class="govuk-body">
-              <%= document.name %><br/>
-              <%= link_to "View in new window", url_for_document(document), target: :_new %>
+              <%= document.name %></br>
             </p>
+            <%= link_to "View in new window", url_for_document(document), target: :_new %>
```

Comment:
Why did this get moved outside the `p` tag?

Reply:
Thanks for the catch - it was an accident :/ Fixed now.